plugins/ingest-attachment/src/test/java/org/opensearch/ingest/attachment/TikaDocTests.java

```diff
@@ -65,6 +65,11 @@ public void testFiles() throws Exception {
 
         try (DirectoryStream<Path> stream = Files.newDirectoryStream(tmp)) {
             for (Path doc : stream) {
+                // Testing change with skipping - will remove in final code
```

Comment:
Should be removed, right?

Reply:
Yes. I skipped some test cases to isolate the failure points. I added `[Do not merge now]` to the title for the same reason. Will revert this.

Reply:
Tika updated now. Reverted this test/temporary change back to original. Resolving this thread.